### PR TITLE
added Dutch translation and minor improvements

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

**Comment by PanderMusubi** [comment]
closes https://github.com/firoorg/firo-site/issues/109

File: _i18n/nl.yml [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+  tos: Servicevoorwaarden
+  newsletter: Nieuws
+  ifsub: Bekijk ons <a href="/blog/">Blog</a> en Telegram-kanaal en blijf op de hoogte van Firo-nieuws.
+  newsbutton: Lees nieuws
+  comm: Communicatie
+  feed: RSS-feed
+
+  
+home:
+  heading: Sparking <br class="brmob">a privacy revolution
```
Comment (erciccione) [inline _i18n/nl.yml:59]: This should be translated too. ignore `<br class="brmob">`

**Review by erciccione — CHANGES_REQUESTED** [review]
Thank you for taking the time to do this :)

- Please add `nl` to the `languages` entry in `_config.yml` so that it will result:

`languages: ["en", "zh-cn", "id", "nl"]`

- We need a Dutch speaker to review the translated strings.

File: _i18n/nl.yml (hunk 1 above)
Comment (PanderMusubi) [reply _i18n/nl.yml]: done in last commit

**Comment by PanderMusubi** [comment]
I have a friend of mine review the translation. Other question, do we have a testing/staging environment for the website so the translation can be viewed in the resulting website? This would also be beneficial for more translations that will be added in the future or major changes which influence translations.

**Comment by justanwar** [comment]
@PanderMusubi We are still trying to decide in #329. 

[A CI file already exists,](https://github.com/firoorg/firo-site/blob/master/.github/workflows/ci.yml) but does not generate previews at the moment.

**Comment by erciccione** [comment]
You can also run the website locally. See https://jekyllrb.com/docs/

File: _i18n/nl.yml [hunk 2]
```diff
@@ -0,0 +1,314 @@
+global:
+  date: '%Y-%m-%d'
+  firo: Firo
+  yeth: "ja"
+  noo: "nee"
+  lelantus: Lelantus
+  title: Firo - Privacybehoudende cryptovaluta
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:7]: Can I use a softhyphen to fix that? If yes, how?

[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:7]: Fixed in another way

**Comment by PanderMusubi** [comment]
> You can also run the website locally. See https://jekyllrb.com/docs/

Thanks, see also https://github.com/firoorg/firo-site/pull/389

**Review by erciccione — APPROVED** [review]
Technically ok, but would be better to have somebody review the translations.

**Comment by PanderMusubi** [comment]
Perhaps find somebody via Discord?

File: _i18n/nl.yml [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+  lelantusdesc: Lelantus is ons privacyprotocol van de volgende generatie dat de hoogste niveaus van praktische privacy biedt zonder vertrouwde opstelling en alleen vertrouwend op solide cryptografische aannames. Lelantus ondersteunt anonimiteitssets die vele malen groter zijn dan andere privacysystemen zonder vertrouwde opstelling.
+  chainlocks: Chainlocks  (gedaan)
+  chainlocksdesc: Chainlocks maken gebruik van Firo's masternodenetwerk om bescherming te bieden tegen 51%-aanvallen. Transacties worden definitief zodra 1 blok is bevestigd.
+  rap: RAP-adressen  (gedaan)
+  rapdesc: RAP-adressen staan een adres toe dat publiekelijk kan worden gedeeld, zodat derden niet kunnen zien hoeveel of wanneer het geld is ontvangen.
+  instant: Instant Send  (gedaan)
+  instantdesc: Instant Send maakt gebruik van Firo's masternodenetwerk om transacties binnen een paar seconden te bevestigen, waardoor het praktisch kan worden gebruikt als ruilmiddel.
+  elysium: Elysium (on Testnet)
+  elysiumdesc: Elysium maakt het mogelijk om tokens (bijvoorbeeld stablecoins, asset backed coins) uit te geven op het netwerk van Firo die profiteren van de privacytechnologie van Firo.
```
Comment (erciccione) [inline _i18n/nl.yml:267]:
```suggestion
  elysiumdesc: Elysium maakt het mogelijk om tokens (bijvoorbeeld stablecoins, asset backed coins) uit te geven op het netwerk van Firo die profiteren van de privacy technologie van Firo.

```

File: _i18n/nl.yml [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+  firopowdesc: FiroPoW is een ProgPOW-variant die is ontworpen voor GPU's en sterk bestand is tegen ASIC's.
+  research: Research
+  lelantussparkaddress: Lelantus Spark sub-addressing and payment proofs (gedaan)
+  lelantussparkaddressdesc: Lelantus Spark ondersteunt subadressen die efficiënter scannen mogelijk maken en betalingsbewijzen waarmee gebruikers kunnen bewijzen dat ze een betaling hebben gedaan.
+  firodex: FiroDEX  (gedaan)
+  firodexdesc: FiroDEX biedt gebruikers de mogelijkheid om van en naar Firo te swappen met behulp van een gedecentraliseerde infrastructuur die gebruik maakt van atomic swap-technologie.
+  bridges: Cross-chain bridges
+  bridgesdesc: Cross chain bridges zullen tokens van andere ecosystemen in en uit Firo's Elysium-tokenlaag laten overbruggen om gebruik te maken van onze privacy-infrastructuur.
+  mixnets: Mixnets
+  mixnetsdesc: Het tunnelen van transacties en knooppuntcommunicatie in mixnets biedt een betere bescherming van metadata, zelfs tegen tegenstanders op staatsniveau. We onderzoeken het Meson-mixnet dat is gebouwd op Katzenpost-softwarebibliotheken.
```
Comment (erciccione) [inline _i18n/nl.yml:284]:
```suggestion
  mixnetsdesc: Het tunnelen van transacties en knooppunt communicatie in mixnets biedt een betere bescherming van metadata, zelfs tegen tegenstanders op staatsniveau. We onderzoeken het Meson-mixnet dat is gebouwd op Katzenpost-softwarebibliotheken.

```

File: _i18n/nl.yml [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+  heading: Merchants
+  predisclaimer: Disclaimer&colon;&#x20;
+  disclaimer: Sommige van deze koppelingen bevatten verwijzingskoppelingen waarbij we een vergoeding kunnen ontvangen wanneer je je aanmeldt of hun producten/diensten gebruikt. Deze vormen geen goedkeuring van het product/de dienst.
+  introduction: Wil je je Firo uitgeven? Hier zijn een aantal plaatsen waar je dat kunt doen! Dit is een onvolledige lijst van plaatsen die Firo accepteren! Als je een bedrijf hebt dat hier vermeld zou willen worden, laat het ons dan weten!
+  button1: Apparel
+  button2: Daten
+  button3: Donatie
+  button4: Freelancing
+  button5: Gaming
+  button6: Gezondsheidsproducten
```
Comment (erciccione) [inline _i18n/nl.yml:301]:
```suggestion
  button6: Gezondheidsproducten

```

File: _i18n/nl.yml [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+  button7: Hosting
+  button8: Juridische diensten
+  button9: Online Marketplaatsen
```
Comment (erciccione) [inline _i18n/nl.yml:304]:
```suggestion
  button9: Online Marktplaatsen

```

File: _i18n/nl.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:7]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

File: _i18n/nl.yml [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+  crowdfunding: Crowdfunding
+  download: Download portemonnees
+  buyfiro: Koop Firo
+  earnfiro: Verdien Firo
+  mine: Mijnen
+  tech: Research
+  team: Team
+  roadmap: Roadmap
+  faq: FAQ
+  fcs: Firo-crowdfundingsysteem
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:40]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

File: _i18n/nl.yml [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+  tos: Servicevoorwaarden
+  newsletter: Nieuws
+  ifsub: Bekijk ons <a href="/blog/">Blog</a> en Telegram-kanaal en blijf op de hoogte van Firo-nieuws.
+  newsbutton: Lees nieuws
+  comm: Communicatie
+  feed: RSS-feed
+
+  
+home:
+  heading: Een vonk <br class="brmob">in de privacyrevolutie
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:59]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

File: _i18n/nl.yml [hunk 9]
```diff
@@ -0,0 +1,314 @@
...
+  heading2: Firo is een privacy cryptovaluta <br>en ecosystem.
+  cta1: Start met Firo
+  cta2: Meer weten
+  about: Hoe Firo werkt
+  aboutp: Firo is uniek in de cryptovalutamarkt omdat het innoveert en baanbrekend is met baanbrekend privacy- en protocolonderzoek. Lees verder en ontdek het zelf.
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:64]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

File: _i18n/nl.yml [hunk 10]
```diff
@@ -0,0 +1,314 @@
...
+  privacy: Privacy
+  privacy1: Firo loopt voorop op het gebied van cryptovalutaprivacy met Lelantus en Lelantus Spark, die betrouwbare, on-chain privacy bieden met hoge anonimiteitssets. De Dandelion++ technologie biedt ook netwerklaagprivacy.
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:66]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

File: _i18n/nl.yml [hunk 11]
```diff
@@ -0,0 +1,314 @@
...
+  avalanchedesc: Avalanche is een consensusalgoritme dat directe transacties, verbeterde beveiliging en fork-free upgrades mogelijk maakt. We onderzoeken Avalanche-consensus als een mogelijke vervanging voor onze masternodes.
+  development: Ontwikkeling
+  heading: Roadmap
+  mnodes: Deterministische Masternodes (gedaan)
+  mnodesq: Long Lived Masternode Quorums  (gedaan)
+  mtpstrip: MTP-strip (gedaan)
+  mtpstripdesc: MTP-strippen reduceert de grootte van Firo's blokketen van 60GB naar 4GB door MTP-bewijzen van oudere blokken te strippen.
+  rebrand: Rebrand  (gedaan)
+  lelantus: Lelantus-activatie  (gedaan)
+  lelantusdesc: Lelantus is ons privacyprotocol van de volgende generatie dat de hoogste niveaus van praktische privacy biedt zonder vertrouwde opstelling en alleen vertrouwend op solide cryptografische aannames. Lelantus ondersteunt anonimiteitssets die vele malen groter zijn dan andere privacysystemen zonder vertrouwde opstelling.
```
[the remark this answers is not in the document]
Comment (PanderMusubi) [reply _i18n/nl.yml:259]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

File: _i18n/nl.yml (hunk 3 above)
Comment (PanderMusubi) [reply _i18n/nl.yml:267]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

File: _i18n/nl.yml (hunk 4 above)
Comment (PanderMusubi) [reply _i18n/nl.yml:284]: Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.

**Comment by PanderMusubi** [comment]
I've processed the feedback. Thanks.

File: _i18n/nl.yml [hunk 12]
```diff
@@ -302,7 +302,8 @@ merchants:
 
   button7: Hosting
   button8: Juridische diensten
-  button9: Online Marketplaatsen
+  button9: Online Marktplaatsen
+
```
Comment (erciccione) [inline _i18n/nl.yml:306]: stray new line

**Review by erciccione — COMMENTED** [review]
Just a minor fix and then looks ready for merge

File: _i18n/nl.yml (hunk 12 above)
Comment (PanderMusubi) [reply _i18n/nl.yml]: Done, were actually two empty lines that needed deleting.

**Review by erciccione — APPROVED** [review]

**Review by justanwar — APPROVED** [review]

**Pull request merged**